src/shared/events/implementations/aws/index.js

```diff
@@ -1,9 +1,49 @@
-function enqueue(topic, payload) {
-    console.log(`aws enqueue: ${topic} ${JSON.stringify(payload, null, 4)}`);
+const AWS = require('aws-sdk');
+
+const sqs = new AWS.SQS();
+const { AWS_REGION, AWS_ACCOUNT_ID } = process.env;
+
+async function enqueue(topic, payload) {
+    const QueueUrl = `https://sqs.${AWS_REGION}.amazonaws.com/${AWS_ACCOUNT_ID}/${topic}`;
```

Comment:
```suggestion
    const queueUrl = `https://sqs.${AWS_REGION}.amazonaws.com/${AWS_ACCOUNT_ID}/${topic}`;
```

local değişkenlerde convention olarak yine camelCase kullanılmış daha önce.